internal/provider/resource_report_graph_query.go

```diff
@@ -0,0 +1,328 @@
+package provider
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"wiz.io/hashicorp/terraform-provider-wiz/internal"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/client"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/utils"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/wiz"
+)
+
+const reportRunStartsAtLayout = "2006-01-02 15:04:05 +0000 UTC"
+
+func resourceWizReportGraphQuery() *schema.Resource {
+	return &schema.Resource{
+		Description: "TBD.",
```

Comment:
Just spotted this TBD value, If you could please update then we can get this PR merged.

Reply:
fixed: ec263330caec91c9d82f351de9e1c01961b53ec5

Reply:
please run `go generate ./... ` to update the docs and push the committed changes

Reply:
done: f3c5f9eff8aeab02f098565652f43a991f567c51